src/components/pages/CompanySubscriptions/CompanySubscriptionsTableColumns.tsx

```diff
@@ -86,6 +87,26 @@ export const CompanySubscriptionsTableColumns = (
       )
   }
 
+  const checkStatus = (row: SubscribedActiveApps) => {
+    return row.status
+  }
+
+  const canShowButton = (row: SubscribedActiveApps) => {
+    const status = checkStatus(row)
+    if (status === SubscriptionStatus.ACTIVE) return true
+    return false
+  }
```

Comment:
we can replace previous nine lines with one line:

`const canShowButton = (row: SubscribedActiveApps) => row.status === SubscriptionStatus.ACTIVE`

Reply:
My bad, checkStatus had multiple conditions wrt app and services but post end point update activity, forgot to remove it.